Flexbox/flex.css

```
@@ -0,0 +1,38 @@
+.header {
+    background-color: orange;
+    display: flex;
+    justify-content: center;
+}
+
+.aside1 {
+    background-color: yellow;
+    width: 250px;
+    height: 80px;
+
+    display: flex;
+    justify-content: center;
+}
```

As sugera ca in loc de aside1 si aside2 sa ai un cod un pic mai reutilizabil:
De asemenea, height-ul poate fi scos
```suggestion
.aside {
    width: 250px;
    display: flex;
    justify-content: center;
}

.left-aside{
     background-color: yellow;
}
.right-aside{
    background-color: pink;
}
```
In HTML ai putea pune

```suggestion
<div class="aside left-aside"></div>
<div class="aside right-aside"></div>
```

Astfel, codul ar fi mai mic si reutilizabil